monitoring.go

```diff
@@ -258,6 +258,44 @@ func (sc *snowflakeConn) rowsForRunningQuery(
 	return nil
 }
 
+// Wait for query to complete from a query id from /queries/<qid>/result endpoint.
+func (sc *snowflakeConn) blockOnRunningQuery(
+    ctx context.Context, qid string) error {
+    resultPath := fmt.Sprintf(urlQueriesResultFmt, qid)
+    resp, err := sc.getQueryResultResp(ctx, resultPath)
```

Comment:
It looks like this only makes one GET request; will this wait until the query is complete, or only for a maximum of 45 seconds?

Reply:
You are correct, I didn't realize that this was the part that leaves after 45 seconds. I'm working on a fix